REQUIRE

```diff
@@ -1,7 +1,8 @@
 julia 0.5
-DataStructures 0.4.3
+DataStructures 0.5.0
 DataFrames 0.9
-NullableArrays 0.0.10
-CategoricalArrays 0.0.6
+NullableArrays 0.1.0
+CategoricalArrays 0.1.0
+AxisArrays 0.0.6
```

Comment:
I'd absolutely like to see better printing of AxisArrays. It's still a young project in terms of the number of hours that have been dedicated to it.

I'm not familiar with any homogenous named array in R, so I'm not entirely sure about the use-case, but the goal of AxisArrays is to provide core support for many disparate domains. Then secondary packages can layer on more domain-specific behaviors.